mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionDaoImpl.java

```diff
@@ -142,7 +142,7 @@ public Future<Optional<JobExecution>> getJobExecutionById(String id, String tena
     Promise<RowSet<Row>> promise = Promise.promise();
     try {
       String jobTable = formatFullTableName(tenantId, TABLE_NAME);
-      String query = format(GET_BY_ID_SQL, jobTable);
+      String query = format(GET_BY_ID_SQL, jobTable).concat("where is_deleted = false");
```

Comment:
Hello @SatishkumarPurswani, GET_BY_ID_SQL query template already contains `WHERE` clause, please add predefined condition `is_deleted = false` to GET_BY_ID_SQL template to fix the query execution. Thank you!